blog/2020-07-01-LetsOptaPlanYourJbpmTasksPart1.adoc

```diff
@@ -0,0 +1,215 @@
+= Let's OptaPlan your jBPM tasks (part 1) - Integrating the two worlds
+:page-interpolate: true
+:awestruct-author: wmedvede
+:awestruct-layout: blogPostBase
+:awestruct-tags: [feature, task assigning, bpm integration]
+
+:awestruct-share_image_filename: CreditDisputeProcessDiagram.png
+
+Traditionally, business process management systems (BPMS) assign human tasks to users on a group basis.
+This information is part of the human tasks configuration.
+
+In the following process, we can see how the Resolve Dispute and Notify Customer tasks are configured with the
```

Comment:
(soft)
Avoid "we" in favour of "you".
Avoid "you" - when possible - in favour of active tense.

For example:
"In the following process, we can see how the Resolve Dispute and Notify Customer tasks are configured ..."
"In the following process, notice how the Resolve Dispute and Notify Customer tasks are configured ..."